shell/app/layout/pages/page-container/page-container.tsx

```diff
@@ -187,7 +187,7 @@ const PageContainer = ({ route }: IProps) => {
           >
             {MainContent}
           </div>
-          <MessageCenter show={showMessage} />
+          {/* <MessageCenter show={showMessage} /> */}
```

Comment:
Does this change related with vite?  or it's a requirement?

Reply:
Nope, comment for test, reverted.